newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -1,160 +1,239 @@
+            disabled
+            index={spritesCount}
+            onAdd={onAddSprite}
+            onEdit={onEditSprites}
+          />,
+        ]}
+      </div>
+    );
+  }
+);
+
+export default class SpritesList extends Component {
+  componentDidMount() {
+    if(!ipcRenderer){return};
+    ipcRenderer.on('piskelSavedChanges', (event, piskelFramePaths) => {
+      const { direction, project } = editedAnimationProp.props;
+      const resourcesManager = project.getResourcesManager();
+      direction.removeAllSprites(); /// clear the old sprite list
+      var i = 0; /// ...We need to recreate it in order to account for any new/removal/reorder frame changes made in piskel
+      for (i = 0; i < piskelFramePaths.length; i++) {
+        var imagePath = piskelFramePaths[i];
+        const imageResource = new gd.ImageResource();
+        imageResource.setFile(imagePath);
+        imageResource.setName(imagePath);
+        resourcesManager.addResource(imageResource);
+        const sprite = new gd.Sprite();
+        sprite.setImageName(imageResource.getName());
+        direction.addSprite(sprite);
+        imageResource.delete();
+        sprite.delete();
+      }
+
+      editedAnimationProp.forceUpdate();
+    });
+  };
+
+  onSortEnd = ({ oldIndex, newIndex }) => {
+    this.props.direction.moveSprite(oldIndex, newIndex);
+    this.forceUpdate();
+  };
+
+  onAddSprite = () => {
+    const {
+      resourceSources,
+      onChooseResource,
+      project,
+      direction,
+    } = this.props;
+    if (!resourceSources) return;
+    const sources = resourceSources.filter(source => source.kind === 'image');
+    if (!sources.length) return;
+
+    onChooseResource(sources[0].name).then(resources => {
+      resources.forEach(resource => {
+        project.getResourcesManager().addResource(resource);
+
+        const sprite = new gd.Sprite();
+        sprite.setImageName(resource.getName());
+        direction.addSprite(sprite);
+      });
+
+      this.forceUpdate();
+    });
+  };
+
+  onEditSprites = () => {
+    if(!electron){
+      alert("This feature is only supported in the Desktop version for now!\nYou can Download it from Gdevelop's website...");
+      return
+    };
+    const { project, direction, resourcesLoader } = this.props;
+    editedAnimationProp = this;
+
+    var imageFrames = []; /// first collect the images to edit
+    for (var i = 0; i < direction.getSpritesCount(); i++) {
+      var spriteImagePath = resourcesLoader.getResourceFullUrl(
+        project,
+        direction.getSprite(i).getImageName()
+      );
+      var importedImage = new Image();
+      importedImage.src = spriteImagePath;
+      spriteImagePath = spriteImagePath.substring(
+        7,
+        spriteImagePath.lastIndexOf('?cache=')
+      );
+      imageFrames.push(spriteImagePath);
+    }
+    const piskelData = {
+      imageFrames: imageFrames,
+      fps: Math.floor(direction.getTimeBetweenFrames() * 480),

```

Comment:
Where is this `480` coming from ? 😁
I think you used the wrong formula:
`fps` means `frame per seconds`. 
`direction.getTimeBetweenFrames()` is the time between frame which is in seconds.
The relation is then: `fps: 1/direction.getTimeBetweenFrames()`. 

You may want to protect again `direction.getTimeBetweenFrames()` being 0 (which should not be the case though):
`fps: direction.getTimeBetweenFrames() > 0 ? 1 / direction.getTimeBetweenFrames() : 1`. 😃

Reply:
ahaha, yeah kind of ugly workaround that was using for testing. Forgot about it. Will use your formula instead xD

Reply:
done